### DISCUSSION TITLE
How to build a single file like Play CDN

### DISCUSSION THREAD

**Discussion by ischanx** [tailwindlabs/tailwindcss#14170] [Help]
Any npm script for this? Build a file like [cdn.tailwindcss.com](https://cdn.tailwindcss.com/)

**Comment by florence0239** [comment]
Hello, [SpotifyPie](https://www.spotify-pie.com)

To create a build process similar to cdn.tailwindcss.com using npm, you can set up a script to compile your Tailwind CSS configuration into a single CSS file. Here’s a step-by-step guide:

Install Tailwind CSS
`npm install -D tailwindcss`
Initialize Tailwind CSS:
`npx tailwindcss init`
Configure Tailwind: Update your tailwind.config.js file to include the paths to your template files.
`/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ["./src/**/*.{html,js}"],
  theme: {
    extend: {},
  },
  plugins: [],
}`
Create a CSS Input File: Create a CSS file (e.g., src/input.css) and add the Tailwind directives.
`@tailwind base;
@tailwind components;
@tailwind utilities;`
Add a Build Script: Update your package.json to include a build script.
`"scripts": {
  "build": "tailwindcss -i ./src/input.css -o ./dist/output.css --minify"
}`
Run the Build Script:
`npm run build`
This will generate a minified CSS file (dist/output.css) that you can use similarly to the Tailwind CDN

Hope this will help you.
Best regards,
florence0239

**Reply by ischanx** [reply]
Thank you for your reply. But i think that it is not a tailwindcss runtime if we execute this script
